### ISSUE TITLE
Azshir the Sleepless double-spawn

### ISSUE THREAD

**Issue by VrogragFishslayer** [slowtorta/turtlewow-bug-tracker#3975]
Azshir the Sleepless, a rarespawn mini-boss in Scarlet Monestary : Graveyard spawned twice in the same instance. 

I only ever saw it once, doesn't mean it can't happen again, but I have tried to no avail. Either it was extraordinary luck, or something in the client is allowing the same rare boss to spawn multiple times. 

NPC: https://database.turtle-wow.org/?npc=6490

I was under the assumption that rare spawn bosses, when there are a number of them could all spawn, but not two of the same one. 
(in case the upload doesn't work, I have an imgur shot too)

https://imgur.com/a/FktxQ9o

![WoWScrnShot_071823_171842](https://github.com/slowtorta/turtlewow-bug-tracker/assets/139925005/22d65a0f-78fc-4ee1-a2d5-15191a4ef585)

Thanks for all the hard work you do. 
<3 Vrograg Fishslayer

**Comment by redmagejoe** [comment]
I assume most are aware of the unique mechanics of Scarlet Monastery Graveyard spawns and thus its rare spawn mechanics, but beyond that, it is indeed not working precisely how online resources suggest it should. The veracity of these claims should be taken with a grain of salt, mind you, but they suggest that once any of the rare spawns die, no more will respawn.

I interpret that to mean that because boss corpses last 2 hours instead of the 15 minutes the rest of SM GY's trash mobs do, that spot is simply occupied and cannot be a spawn point for another rare or trash mob. This would be very difficult to prove if it's "working as intended" or not, but it may be worth writing this off as just the way it is. That said, it is strange that two of the same rare can spawn AT THE SAME TIME, but there may be no easy way without scripting to prevent this from happening.

tl;dr it's hard to know if this is a bug or not

**Comment by VrogragFishslayer** [comment]
I had that feeling too, but I made a report in case something was changed behind the scene and this unintended side effect is going on. Also since SM:GY is farmed for gravemoss, thought more people might have witnessed this too. I have tried to replicate it so far but aside from a quicker way to test lockouts I don't think I'll be able to ever replicate it again. 

(Thanks for your contributions Redmagejoe, great work on the resource/threat gain thread I covered)

**Comment by azolom** [comment]
I also farm gm at gy, and have seen double azshir spawns twice. I don't think I've ever seen a duplicate of another boss. Will also note that classic has more gm nodes -- not sure about vanilla -- but my assumption is that this is intentional.
